recipes/libzippp/all/conanfile.py

```diff
@@ -75,7 +83,6 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = tools.collect_libs(self)
-        self.cpp_info.names["cmake_find_package"] = "libzippp"
-        self.cpp_info.names["cmake_find_package_multi"] = "libzippp"
+        self.cpp_info.set_property("cmake_file_name", "libzippp")
```

Comment:
Please restore the 2 lines above

Reply:
Sorry. I forget to restore. 
i fixed it.